python-lib/symspell_checker.py

```diff
@@ -344,22 +353,23 @@ def _format_output_df(self, df: pd.DataFrame) -> None:
 
         Args:
             df: Input pandas DataFrame
+
         """
-        df.drop(self._tokenizer.tokenized_column, axis=1, inplace=True)
-        corrected_text_column = list(self.output_column_description_dict.keys())[0]
-        spelling_mistakes_column = list(self.output_column_description_dict.keys())[1]
-        misspelling_count_column = list(self.output_column_description_dict.keys())[2]
+        df.drop(self.tokenizer.tokenized_column, axis=1, inplace=True)
+        corrected_text_column = list(self.output_column_descriptions.keys())[0]
```

Comment:
can you call list(self.output_column_descriptions.keys()) once ?

Reply:
I could, it's honestly a very minor optimization. `output_column_description_dict` is a small dictionary, it doesn't cost much to ask for keys.

Reply:
Agreed